.azure-pipelines/azure-pipelines-merge.yml

```diff
@@ -191,42 +191,46 @@ jobs:
       scriptSource: 'filePath'
       scriptPath: 'scripts/findEmptyHelpTexts.py' 
 
-# - job: 'Run_Test_From_Old_Release'
-#   dependsOn : Build_Publish_Azure_CLI_Test_SDK
-#   pool:
-#     vmImage: 'macOS-10.13'
-
-#   steps:
-#   - script: git checkout release-0.10.0
-
-#   - template: templates/run-tests.yml
-#     parameters:
-#       pythonVersion: '3.x'
-#       runOnlyRecordedTests: 'true'
-
-# - job: 'Check_Back_Compat_Arguments'
-#   pool:
-#     vmImage: 'vs2017-win2016'
-
-#   steps:
-#   - task: UsePythonVersion@0
-#     inputs:
-#       versionSpec: '3.x'
-#       architecture: 'x64'
-
-#   - template: templates/setup-ci-machine.yml
-
-#   - template: templates/install-azure-cli-edge.yml
-
-#   - script: 'python setup.py sdist bdist_wheel'
-#     displayName: 'Build wheel for Azure DevOps CLI extension'
-#     workingDirectory: 'azure-devops/'
-
-#   - task: PythonScript@0
-#     displayName: 'Run Back Compat Argument Check'
-#     inputs:
-#       scriptSource: 'filePath'
-#       scriptPath: 'scripts/backCompatChecker.py' 
+- job: 'Run_Test_From_Old_Release'
+  dependsOn : Build_Publish_Azure_CLI_Test_SDK
+  pool:
+    vmImage: 'macOS-10.13'
+
+  steps:
+  - script: git checkout release-0.11.0
+
+  # Delete test files which are broken by design
+  - script: rm tests/test_devopsUserTest.py
```

Comment:
Tracking item to enable this?

Reply:
https://github.com/Azure/azure-devops-cli-extension/issues/722